new upstream release -- 0.20.0

Ast3risk-ops opened the pull request:
fixes #7

Ast3risk-ops committed:
new upstream release -- 0.20.0

barnumbirr commented:
Thank you for the PR. I'm on mobile (or I would have pushed the release myself) but LGTM apart from my comment in `debian/changelog`. I'll merge as soon as I can once you push the fix.

Ast3risk-ops commented:
> Thank you for the PR. I'm on mobile (or I would have pushed the release myself) but LGTM apart from my comment in `debian/changelog`. I'll merge as soon as I can once you push the fix.

I don't see the comment, what's the issue?

barnumbirr commented:
> I don't see the comment, what's the issue?

Github Mobile is weird. It's nothing major but the year is set to 2024 in `debian/changelog`.

Ast3risk-ops committed:
fix changelog year

Ast3risk-ops commented:
> > I don't see the comment, what's the issue?
> 
> Github Mobile is weird. It's nothing major but the year is set to 2024 in `debian/changelog`.

Thought I changed that already, fixed.

barnumbirr commented:
Merged! Thanks.

Ast3risk-ops commented:
> Merged! Thanks.

Just realized it's Saturday instead of Friday in the changelog, deleted my fork already 🤦 

Also, the build action hasn't run yet.

barnumbirr commented:
Ah, missed that too. I'm in the middle of moving to a new place so my head is not on straight.
I'm trying to create a new tag on mobile so the build runs.

barnumbirr commented:
Github mobile doesn´t allow tag creation but I've managed to find a laptop. Action is running, `.deb` should be live soon.
Cheers.
